test-runner/build.sbt

```diff
@@ -74,11 +74,10 @@ mainClass := Some("com.typesafe.spark.test.mesos.framework.runners.MesosIntegrat
 val mainDCOSClass = "com.typesafe.spark.test.mesos.framework.runners.DCOSIntegrationTestRunner"
 
 //invoking inputtasks is weird in sbt. TODO simplify this
-def runMainInCompile(sparkHome: String,
-                     mesosMasterUrl: String,
+def runMainInCompile(args: Seq[String],
                      applicationJar: String) = Def.taskDyn {
-
- val main = s"  ${mainClass.value.get} $sparkHome $mesosMasterUrl $applicationJar"
```

Comment:
:) It's very inconsistent in the code space, I think we should skip the empty line in the beginning, unless we want to add it in every method.